model/ConceptProperty.php

```diff
@@ -56,14 +59,26 @@ public function getLabel()
     }
 
     /**
-     * Returns a gettext translation for the property tooltip.
+     * Returns text for the property tooltip.
      * @return string
      */
     public function getDescription()
     {
         $helpprop = $this->prop . "_help";
 
-        return gettext($helpprop); // can't use string constant, it'd be picked up by xgettext
+        // see if we have a translation with the help text
+        $help = gettext($helpprop);
```

Comment:
This will prefer Skosmos translations, but that will must likely be OK.

Reply:
Yes, that's the intent (same as for property labels)